ocs_ci/ocs/resources/storage_cluster.py

```diff
@@ -853,6 +854,27 @@ def osd_encryption_verification():
         log.info("Luks header info found for all the encrypted osds")
 
 
+def in_transit_encryption_verification():
+    """
+    Verify in-transit encryption is enabled.
+    """
+    toolbox = pod.get_ceph_tools_pod(skip_creating_pod=True)
```

Comment:
would be good to add a log massage saying that in transit encryption is about to be validated

Reply:
Added log message when in-transit encryption verification process.